pkg/config/config.go

```diff
@@ -29,17 +30,31 @@ var (
 	TEST_TERMINAL_WIDTH = "TEST_TERMINAL_WIDTH"
 )
 
+// Profile storage drivers
+const (
+	_ = iota
+	ProfileStoreInMemory
+	ProfileStoreFile
+	ProfileStoreNativeKeyring
+)
+
 type Output struct {
 	Format string `yaml:"format" default:"styled"`
 }
 
 type Config struct {
 	Output Output `yaml:"output"`
+	// TODO: make this actually configurable by flags/env
```

Comment:
The config yaml is being deprecated soon, and we need to decide how we want to handle storage driver selection in the CLI. Some options:
1. if Windows or Linux, always file. If Mac, always keychain.
2. set with environment.
3. set with global flags.